packages/webpack/src/loader.ts

```diff
@@ -70,10 +79,7 @@ export default async function vocabLoader(this: LoaderContext) {
     throw new Error(`Webpack didn't provide an async callback`);
   }
 
-  // @ts-expect-error We define our own loader context type, getOptions expects
-  // webpack's LoaderContext type, but it's missing the target field
-  // https://github.com/webpack/webpack/issues/16753
-  const config = getOptions(this) as unknown as UserConfig;
```

Comment:
This fix was only released in webpack 5.76, does this style type check with wepack<5.75?

Reply:
It works with the Webpack installed in this project. But good catch, the peer dependency should be set to `^5.37.0` https://github.com/webpack/webpack/pull/13164

Reply:
But we use `this.target` a few lines down, that would still fail to typecheck below 5.76 right? Don't really want to set a 5.76 peer dep on this package, though we could do it with a sku update i suppose.

Reply:
Just using it as a dev dep, 5.76 isn't required for consumers 🤦